skimage/restoration/deconvolution.py

```diff
@@ -365,13 +365,22 @@ def richardson_lucy(image, psf, iterations=50, clip=True):
     ----------
     .. [1] http://en.wikipedia.org/wiki/Richardson%E2%80%93Lucy_deconvolution
     """
+    direct_time = lambda n, m, k, l: k*l * n*m
+    def fft_time(m, n, k, l):
+        return m*np.log(m) + n*np.log(n) + k*np.log(k) + l*np.log(l)
+
+    time_ratio = 71.468 * fft_time(*image.shape, *psf.shape)
```

Comment:
```
time_ratio = 71.468 * fft_time(*tuple(list(image.shape) + list(psf.shape)))
```
solves the issue. I'm not sure you're allowed to unpack two tuples in a list of arguments.